src/gval/accessors/gval_array.py

```diff
@@ -18,3 +21,34 @@ class GVALArray(GVALXarray):
 
     def __init__(self, xarray_obj: xr.DataArray):
         super().__init__(xarray_obj)
+
+    def cat_plot(
+        self,
+        title: str = "Categorical Map",
+        colormap: str = "viridis",
+        figsize: Tuple[int, int] = (6, 4),
+        legend_labels: list = None,
+    ):
+        """
+        Plots categorical Map for xarray dataset
+
+        Parameters
+        __________
+        title : str
```

Comment:
It would be good to reference the matplotlib/xarray documentation that these arguments wrap to.